boards/nxp/mimxrt595_evk/board.c

```diff
@@ -279,34 +279,32 @@ static int mimxrt595_evk_init(void)
 	pm_policy_state_lock_get(PM_STATE_SUSPEND_TO_IDLE, PM_ALL_SUBSTATES);
 
 #ifdef CONFIG_I2S
-
+#ifdef CONFIG_I2S_TEST_SEPARATE_DEVICES
```

Comment:
This Kconfig option is only used by I2S tests. It's quite weird that it is handled in some board initialization code.

Reply:
@anangl It's true that this property could be named differently (as I didn't author that name), I believe it has its place in either soc.c or board.c. This property implies how the signals between Flexcomm peripherals in an I2S mode should be shared. Duplex audio on the RTxxx devices with a single codec is handled by configuring one peripheral as an I2S master and the other as a slave. The slave peripheral then has the LRCK (WS) and SCK signals as inputs. It may be desirable not to set up signal sharing, for example when these devices are used with separate ADCs and DACs.